schemas/common/selfservice.schema.json

```diff
@@ -0,0 +1,170 @@
+          "format": "uri"
+        },
+        "deployment": {
+          "name": "Deployment",
+          "description": "Deployment is the final act of deploying an application into an environment prior to going live in that enviroment. This should be achieved via an automated process working from specifcation and configuration stored in the codeRepository. Some applications may require the installation of packages directly into a service or API provided by the application. Where this is the case the deployment process must be capable of supporting that style of deployment and must not make the application or any of its parts available to serve requests while the deployment is incomplete. The deployment process should include stepts to validate that the depoloyment was sucessfull. Deployment is not the same as provisioning, which is a pre-requisite for deployment. Typically a deployment will be perfomed every time changes in the code repository needs to be made available in the environment. Typically the deployment process is implemented by the ciCdPipeline",
+          "type": "string",
+          "format": "uri"
+        },
+        "artifact": {
+          "name": "Artifact",
+          "description": "An Artifact use a package forming the implementation of a service. It is is stored in the artifactRepository publised to the artifactRepository by the ciCdPipeline. An artifact has a version number. Once an artifact with a version number is published it becomes immutable, subject to the versioning policy of the artifactRepository.",
+          "type": "string",
+          "format": "uri"
+        },
+
+        "artifactRepository": {
+          "name": "Artifact Repository",
+          "description": "An Artifact repository stores released binaries either from vendors (including Adobe) or created by teh ciCdPipeline. These artifacts are used by the deployment pipleine to deploy an application into an environment. The artifactRepository will allow many versions of an artifact to exist and will be organised in a way that will allow artifacts specific to a tenant, application, topology and environment to be stored. The artifact repsitory must not be used to store any sensitive data.",
+          "type": "string",
+          "format": "uri"
+        },
+        "selfServiceApi": {
+          "name": "Self Service API",
+          "description": "The Self Serivce API is a well defined API that will, using the model defined in XDM expose to a suitably authorised client a service implemenation that allows that client to one or more applications within a tenant. The API must maintain tenant boundaries, as defined and controlled by IMS. The API will expose functionality based on managing a multi tenant application, and some will have access to less functionality (eg a end Customer of Adobe managing their own single tenant deployment.)",
+          "type": "string",
+          "format": "uri"
+        },
+        "selfServicePortal": {
```

Comment:
We have `manageUI` somewhere in assets, so I'd keep this consistent `selfServiceUI`